### ISSUE TITLE
Question on NLI predict scores

### ISSUE THREAD

**Issue by Tolga28A** [UKPLab/sentence-transformers#1819]
Based on this NLI implementation: 

https://huggingface.co/cross-encoder/nli-deberta-v3-base#:~:text=from%20sentence_transformers%20import,axis%3D1)%5D

model.predict returns an array like below for each pair:

[-2.0733533 , -3.373496  ,  5.690237]

How can we scale or use these numbers for a simple threshold optimization? For example how can I increase the threshold for a label (let's say 'contradiction') to decrease the false positives? I thought we could apply a basic threshold value for the maximum score for a pair but since these scores are not scaled between [0-1], I could not be sure how to do this...

Thanks!

**Comment by nreimers** [comment]
You can softmax normalize them, then all scores are 0-1

**Comment by Tolga28A** [comment]
Thanks for the quick answer and also for this great package @nreimers.

**Comment by Matthieu-Tinycoaching** [comment]
@nreimers Could this softmax be applied a posteriori on logits scores?
